packages/fxa-auth-server/test/local/senders/email.js

```diff
@@ -42,9 +45,18 @@ const MESSAGE = {
   primaryEmail: 'c@d.com',
   productId: 'wibble',
   planId: 'plan-example',
-  productName: 'example product',
+  productName: 'Firefox Fortress',
+  planName: 'Firefox Fortress Monthly',
```

Comment:
Same as other comment: `planName` can be removed, since we are using `productName`